test/server_test.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
         self.unix_socket = os.path.join(self.temp_dir, 'luigid.sock')
 
     def run_server(self):
-        luigi.server.run(unix_socket=unix_socket)
+        luigi.server.run(unix_socket=self.unix_socket)
```

Comment:
I have no idea about that.

Original message was:
```
test/server_test.py:102:38: F821 undefined name 'unix_socket'
```

I just fixed that code. But if to look deeper - you can find that was dead code.

`UNIXServerClient` used by `UNIXServerTestRun` which in its turn have only `tearDown` method.